### DISCUSSION TITLE
关于IC2中<橡胶靴>无法正常减免<摔落伤害>且无<耐久值>和<防御值>bug

### DISCUSSION THREAD

**Discussion by WaterDouble** [IndustrialCraftMC/IndustrialCraft2#39] [【漏洞反馈】Bugs Report]
使用模组版本：1.industrialcraft-2-2.9.84+ex119-1.19.2-forge
2.[1.19.2]Jei物品管理器(forge11.3.0.262)

游戏版本：1.19.2 forge-43.2.0 Op-I1-pre4

问题描述：
之前版本的减免规则如下：
在10格以内不会损伤生命值，

11~18格损伤1格生命值;

19~26格损伤2格生命值;

27~34格损伤3格生命值;

35~42格损伤4格生命值;

43~50格损伤5格生命值;

51~58格损伤6格生命值;

59~66格损伤7格生命值;

67~74格损伤8格生命值;

75~82格损伤9格生命值;

90格死亡。

但是现在穿上后<摔落伤害>却是正常<裸装>的<摔落伤害>，并未得到任何减免。且其好似无<耐久值>和<防御值>

**Comment by AbbottMc** [comment]
请在一条对话中只反馈一个问题，如果有多个问题的话，请分别新建对话反馈

**Reply by WaterDouble** [reply]
okk

**Comment by AbbottMc** [comment]
衰落伤害的问题从1.12.2的版本起就是大于八格不会吸收伤害了，所以理论上不算bug。
不过之后有没有可能复刻相关特性有待讨论（毕竟超过八格就无效了还是有点拉的）
